app/_src/gateway/upgrade-v2.md

```diff
@@ -20,10 +20,10 @@ distinction between major, minor, and patch versions.
 The upgrade to {{page.kong_version}} is a **minor** upgrade.
 
 {:.important}
-> **Important**: Blue-green migration in traditional mode for versions below 2.8.2 to 3.0.x is not supported.
+> **Important**: Blue-green migration in traditional mode for versions below 2.8.x to 3.0.x is not supported.
 The 2.8.2 release includes blue-green migration support. If you want
 to perform migrations for traditional mode with no downtime,
-upgrade to 2.8.2, [then migrate to {{page.kong_version}}](#migrate-db).
+upgrade to 2.8.x, [then migrate to {{page.kong_version}}](#migrate-db).
```

Comment:
This isn't exactly true. The reason this version said `2.8.2` and not `2.8.x` is because versions before 2.8.2 _will not work_. So it should be something like this:

```suggestion
upgrade to a 2.8.x version (at least 2.8.2), [then migrate to {{page.kong_version}}](#migrate-db).
```